gamemode/core/meta/sh_item.lua

```diff
@@ -45,39 +45,60 @@ function ITEM:GetID()
 	return self.id
 end
 
+--- Returns the name of the item.
+-- @realm shared
+-- @treturn string The name of the item
 function ITEM:GetName()
 	return (CLIENT and L(self.name) or self.name)
 end
 
+--- Returns the description of the item.
+-- @realm shared
+-- @treturn string The description of the item
 function ITEM:GetDescription()
 	if (!self.description) then return "ERROR" end
 
 	return L(self.description or "noDesc")
 end
 
+--- Returns the model of the item.
+-- @realm shared
+-- @treturn string The model of the item
 function ITEM:GetModel()
 	return self.model
 end
 
+--- Returns the skin of the item.
+-- @realm shared
+-- @treturn number The skin of the item
 function ITEM:GetSkin()
 	return self.skin or 0
 end
 
+--- Deprecated function which currently returns nothing.
+-- @realm shared
+-- @treturn nil Nothing
 function ITEM:GetMaterial()
```

Comment:
Probably shouldn't add documentation for deprecated methods

Reply:
Done